.yamllint.yml

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: MIT
 ---
-extends: yamllint_defaults.yml
+extends: default
```

Comment:
what is the reason switching from `extends: yamllint_defaults.yml` into `extends: default` ?

Reply:
We're getting rid of `yamllint_defaults.yml` and we'll just use the built-in defaults in `default`